t/rose.config_tree/00-self.t

```diff
@@ -17,4 +17,5 @@
 # You should have received a copy of the GNU General Public License
 # along with Rose. If not, see <http://www.gnu.org/licenses/>.
 #-------------------------------------------------------------------------------
-exec python3 $ROSE_HOME/lib/python/rose/config_tree.py "$@"
+from metomi.rose.config_tree import _Test
+_Test().run()
```

Comment:
Pop a newline at the end of this file, to avoid nastiness that can sometimes occur without.